apps/files/src/components/FileLink.vue

```diff
@@ -1,32 +1,35 @@
-    <FileLinkForm v-if="formOpen" v-bind:params="params" :linkId="linkId" />
-    <div class="uk-text-right" v-if="!formOpen">
-      <oc-button v-if="!linksLoading" variation="primary" icon="add" @click="$_openForm()" v-translate>Add Link</oc-button>
+    <div class="uk-text-right">
+      <oc-button v-if="!linksLoading" :disabled="(formOpen || linkId)" variation="primary" icon="add" @click="$_openForm()" v-translate>Add Link</oc-button>
       <button v-else disabled class="uk-button uk-button-default uk-position-relative"><oc-spinner class="uk-position-small uk-position-center-left" size="small" /><span class="uk-margin-small-left" v-translate>Loading</span></button>
     </div>
+    <FileLinkForm v-if="formOpen" v-bind:params="params" :linkId="linkId" class="uk-margin" />
     <transition-group tag="ul" name="custom-classes-transition" enter-active-class="uk-animation-slide-left-medium" leave-active-class="uk-animation-slide-right-medium uk-animation-reverse" class="uk-list uk-list-divider uk-overflow-hidden">
       <li v-for="(link, index) in $_links" :key="'li-' + index">
         <oc-grid flex gutter="small">
           <div class="uk-width-auto">
-            <oc-icon v-if="link.password" name="lock" class="uk-icon-button" />
-            <oc-icon v-else name="link" class="uk-icon-button" />
+            <oc-icon v-if="link.password" name="lock" size="medium" />
+            <oc-icon v-else name="link" size="medium" />
           </div>
-          <div class="uk-width-expand">
+          <div class="uk-width-expand uk-text-truncate">
             <span class="uk-text-bold">{{ link.name }}</span><br>
-            <span class="uk-text-meta">{{ link.description }}<template v-if="link.expiration"> | <span v-translate>Expires</span> {{ formDateFromNow(link.expiration) }}</template></span>
+            <a :href="link.url" :uk-tooltip="$_tooltipTextLink" class="uk-text-small uk-link">{{ link.token }}</a> <span class="uk-text-meta">| {{ link.description }}<template v-if="link.expiration"> | <span v-translate>Expires</span> {{ formDateFromNow(link.expiration) }}</template></span>
           </div>
           <div class="uk-width-auto uk-button-group">
-            <oc-button icon="edit" @click="$_editLink(link)"/>
-            <oc-button icon="delete" @click="$_removeLink(link)" />
+            <oc-button :disabled="(formOpen || linkId)" icon="edit" @click="$_editLink(link)"/>
+            <oc-button :disabled="(formOpen || linkId)" icon="delete" @click="$_removeLink(link)" />
             <button class="uk-button uk-button-default uk-position-relative" @click.stop="$_copyToClipboard(link, $event)">
-              <oc-icon name="link" class="uk-position-center" />
+              <oc-icon name="copy_to_clipboard" class="uk-position-center" />
             </button>
           </div>
         </oc-grid>
         <FileLinkForm v-if="linkId === link.id" class="uk-margin-top" v-bind:params="params" :context="'edit'" :linkId="linkId"/>
```

Comment:
Remove pls `v-bind` before the params - https://vuejs.org/v2/style-guide/#Directive-shorthands-strongly-recommended